CONTRIBUTING.md

```diff
@@ -51,6 +53,7 @@ will be closed. We need to ensure the code we're adding to master is written to
 * **Do** use the many module mixin [API]s.
 * **Do** include instructions on how to setup the vulnerable environment or software.
 * **Do** include [Module Documentation] showing sample run-throughs.
+* **Do** ask cve@rapid7.com for a CVE ID if this is describes a new vulnerability (remember to mention your PR number!)
```

Comment:
```suggestion
* **Do** ask cve@rapid7.com for a CVE ID if this describes a new vulnerability (remember to mention your PR number!)
```

Reply:
Will go ahead and apply this fix. Nice spotting @smcintyre-r7 :)